app/controllers/catalog_controller.rb

```diff
@@ -64,7 +64,7 @@ def index
     }
 
     # solr path which will be added to solr base url before the other solr params.
-    # config.solr_path = 'select'
+    config.solr_path = 'search'
```

Comment:
This is how this setting really ought to have always been. We use a custom handler found at /search, we should specify it here. I think Solr was working because in our solr config we specify in the `search` handler that it is default, but this to me is clearer, and less working-by-accident. Haven't look at this in a while though so please let me know if I am forgetting/missing something!

Reply:
yes, since `/search` handler has default set up to it, it was working but having it explicitly set here is nice and clear, I agree 👍